samcli/commands/build/command.py

```diff
@@ -124,6 +124,20 @@
     type=click.Path(),  # Must be a json file
     help="Path to environment variable json file (e.g., env_vars.json) to pass into build containers",
 )
+@click.option(
+    "--build-image",
+    "-bi",
+    default=None,
+    multiple=True,  # Can pass in multiple build images
+    required=False,
+    help="Docker image URIs for building functions."
+    "You can specify for each individual function with "
+    "(--build-image FunctionLogicalID=public.ecr.aws/sam/build-nodejs14.x:latest)."
+    "You can specify for all functions with just the image URI "
+    "(--build-image public.ecr.aws/sam/build-nodejs14.x:latest)."
+    "A combination of the two can be used. If a function does not have build image specified or"
+    "a image uri for all functions, the default SAM CLI build images will be used.",
```

Comment:
- Needs space after `or`. Same for L133, L135 and L137.
- typo: image URI

Reply:
Fixed.